app/routes/admin/email/components/EmailLists.tsx

```diff
@@ -66,8 +66,9 @@ export default class EmailLists extends Component<Props> {
           brukere eller grupper. Lister kan ikke slettes, men mottakere kan
           endres. Delen av adressen som kommer før @abakus.no er unik og kan
           ikke brukes andre steder på abakus.no. Lister er åpne og alle kan
-          sende epost til disse. Ønsker brukere å sende mail fra en @abakus.no
-          adresse må de få opprettet en personlig adresse under Brukere.
+          sende e-post til disse. Ønsker brukere å sende e-post fra en
+          @abakus.no adresse må de få opprettet en personlig adresse under
+          Brukere.
```

Comment:
```suggestion
          sende e-post til disse. Ønsker brukere å sende e-post fra en
          @abakus.no-adresse må de få opprettet en personlig adresse under
          Brukere.
```

Reply:
Same as above. I'm just not sure lol. I guess since it is custom to write "IP-adresse". I'll note it.